Pod/Classes/GDGBelongsToRelation.m

```diff
@@ -55,12 +55,11 @@ - (void)fill:(NSArray<GDGEntity *> *)entities withProperties:(NSArray<NSString *
 	}
 }
 
-- (NSString *)joinCondition
+- (NSString *)joinConditionForSource:(GDGSource *)source withSource:(GDGSource *)joinedSource
```

Comment:
I think it would be a better name here something like:

`-joinConditionFromSource:toSource:`
or
`-conditionForSource:joinWithSource:`
or yet
`conditionForJoinSource:withSource:`

The way it is right now, is not clear what does this method should be signed for